runtest-cluster

```diff
@@ -8,7 +8,7 @@ done
 
```

Comment:
This file seems to have been committed with CRLF endings, leading to "/bin/sh^M: bad interpreter" errors. This can be fixed by running `set fileformat=unix` in vim and saving.